compysition/actor.py

```diff
@@ -186,16 +186,15 @@ def send_event(self, event, queues=__NOT_DEFINED, check_output=True):
         """
 
         if queues is self.__NOT_DEFINED:
```

Comment:
It is generally more pythonic to use None as opposed to a sentinel value such as __NOT_DEFINED. You can then alter your `if` statement above as commented.